src/app/components/core/user-management/update-user/update-user.component.ts

```diff
@@ -28,38 +31,41 @@ export class UpdateUserComponent implements OnInit, OnChanges {
     this.userRecord = {
       userId: 0,
       username: '',
+      description: '',
       password: '',
+      real_name: '',
```

Comment:
why this, access_method,role_id  in snake_case vs others?

Reply:
It is same as API param. We need to keep these same here.